generate-bookmarks-html

```diff
@@ -72,7 +72,7 @@ EOD
 
         # Use git log to find the timestamps for the initial and last commits.
         unset add_date last_modified
-        while read timestamp subject
+        while read -r timestamp subject
```

Comment:
Good one. `read -r` is what I type by default nowadays. Might have been influenced by this very change. :-)

I think you meant “backslashes” instead of “backspaces”, unless you have some very funky bookmarklet titles.